examples/twr_aloha/README.md

```diff
@@ -0,0 +1,70 @@
+## Decawave TWR_ALOHA Example
+
+This example allows testing different two-way ranging algorithm between
```

Comment:
```suggestion
This example aims at testing different two-way ranging algorithms between
```

Reply:
Will correct, `algorithms`, I don't agree with the rewording.